oak-run-elastic/src/main/java/org/apache/jackrabbit/oak/run/Main.java

```diff
@@ -0,0 +1,35 @@
+package org.apache.jackrabbit.oak.run;
+
+import org.apache.jackrabbit.oak.run.commons.Command;
+import org.apache.jackrabbit.oak.run.commons.Utils;
+
+import java.util.Locale;
+
+import static java.util.Arrays.copyOfRange;
+import static org.apache.jackrabbit.oak.run.AvailableElasticModes.MODES;
+
+public final class Main {
+    private Main() {
+        // Prevent instantiation.
+    }
+
+    public static void main(String[] args) throws Exception {
```

Comment:
While it works, isn't the "new" way using "String... args"?

Reply:
ah..I still am in the habit to use String[] .. changed it to String... args now